drivers/pulseaudio/audio_driver_pulseaudio.cpp

```diff
@@ -47,6 +47,7 @@ void AudioDriverPulseAudio::pa_state_cb(pa_context *c, void *userdata) {
 		case PA_CONTEXT_READY:
 			ad->pa_ready = 1;
 			break;
+		default: {}
```

Comment:
```
drivers/pulseaudio/audio_driver_pulseaudio.cpp:41:9: warning: enumeration value 'PA_CONTEXT_AUTHORIZING' not handled in switch [-Wswitch]
drivers/pulseaudio/audio_driver_pulseaudio.cpp:41:9: warning: enumeration value 'PA_CONTEXT_CONNECTING' not handled in switch [-Wswitch]
drivers/pulseaudio/audio_driver_pulseaudio.cpp:41:9: warning: enumeration value 'PA_CONTEXT_SETTING_NAME' not handled in switch [-Wswitch]
drivers/pulseaudio/audio_driver_pulseaudio.cpp:41:9: warning: enumeration value 'PA_CONTEXT_UNCONNECTED' not handled in switch [-Wswitch]
```
@marcelofg55

Reply:
I would add a TODO note there to check if we should handle PA_CONTEXT_UNCONNECTED, in the very rare case that PulseAudio server gets killed or something like that (I haven't tested what happens if this happens though).
Besides that it's okay.

Reply:
Thanks, will do.